tensorflow/contrib/go/session_test.go

```diff
@@ -0,0 +1,159 @@
+package tensorflow_test
+
+import (
+	"testing"
+
+	tf "github.com/tensorflow/tensorflow/tensorflow/contrib/go"
+)
+
+func TestNewSession(t *testing.T) {
+	graph, err := tf.LoadGraphFromTextFile("test_data/tests_constants_outputs.pb")
+	if err != nil {
+		t.Fatal(err)
+	}
+	s, err := tf.NewSession()
+	if err := s.ExtendGraph(graph); err != nil {
+		t.Fatal(err)
+	}
+	outputs := []string{
+		"output1",
+		"output2",
+	}
+
+	output, err := s.Run(nil, outputs, nil)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	if len(output) != len(outputs) {
+		t.Fatal("There was", len(outputs), "expected outputs, but:", len(output), "obtained")
+	}
+}
+
+func TestInputParams(t *testing.T) {
+	inputSlice1 := []int64{1, 2, 3}
+	inputSlice2 := []int64{3, 4, 5}
+
+	t1, err := tf.NewTensor(inputSlice1)
+	if err != nil {
+		t.Error("Problem trying create a new tensor, Error:", err)
```

Comment:
t.Fatal("Error creating a new tensor:", err)
(apply throughout this file - both the phrasing change and the switch from error+FailNow to Fatal or Fatalf)